src/Aspire.Dashboard/Extensions/TelemetryFilterFormatter.cs

```diff
@@ -23,7 +23,13 @@ private static string SerializeFilterToString(TelemetryFilter filter)
             _ => null
         };
 
-        return $"{Escape(filter.Field)}:{condition}:{Escape(filter.Value)}";
+        var filterString = $"{Escape(filter.Field)}:{condition}:{Escape(filter.Value)}";
+        if (!filter.Enabled)
+        {
+            filterString += ":disabled";
```

Comment:
nit: move "disabled" to a const. Use and here in the detection check
```suggestion
            filterString += $":{DisabledText}";
```